projects/angular/src/data/datagrid/_datagrid.clarity.scss

```
@@ -2019,148 +1189,75 @@
 
   .datagrid-detail-body {
     flex: 1 1 auto;
-    @include mixins.css-var(
-      padding,
-      clr-datagrid-detail-body-padding,
-      0 variables.$clr_baselineRem_1,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      color,
-      clr-datagrid-detail-body-text-color,
-      variables.$clr-p1-color,
-      variables.$clr-use-custom-properties
-    );
+    padding: datagrid-variables.$clr-datagrid-detail-body-padding;
+    color: datagrid-variables.$clr-datagrid-detail-body-text-color;
   }
 
   .datagrid-detail-header {
     flex: 0 0 auto;
-    font-size: variables.$clr_baselineRem_0_75;
-    line-height: variables.$clr_baselineRem_1_4;
-    @include mixins.css-var(
-      padding,
-      clr-datagrid-detail-header-padding,
-      0 0 0 variables.$clr_baselineRem_1,
-      variables.$clr-use-custom-properties
-    );
+    @include mixins.generate-typography-token('SECTION-20-STD');
+    padding: datagrid-variables.$clr-datagrid-detail-header-padding;
```

Same as body padding.
```suggestion
    padding: 0;
```